app/models/user.rb

```diff
@@ -3,4 +3,5 @@ class User < ActiveRecord::Base
   # :confirmable, :lockable, :timeoutable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :trackable, :validatable
+  has_many :restaurants
```

Comment:
we should probably nullify the restaurants author if we destroy the user. checkout the `:nullify` option for when you define a destroy dependency